client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -25,14 +25,14 @@ class SimplePaymentsView extends Component {
 			return ( <QuerySimplePayments siteId={ siteId } productId={ productId } /> );
 		}
 
-		const { title, description, price, currency } = product;
+		const { title, description, price, currency, multiple } = product;
+
+		const paypalButtonImageUrl = '/calypso/images/simple-payments/preview-paypal-button-gray.png';
+		const multipleIsEnabled = Number( multiple ) !== 0;
```

Comment:
The type of `multiple` should be a boolean. The current values are a bit messy -- I see `""`, `"0"` or `"1"`. What are the values of the post metadata key `spay_multiple` supposed to be? If there is any conversion between them needed, it should be in the `customPostToProduct` and `productToCustomPost` functions in the [data layer code](https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/wpcom/sites/simple-payments/index.js#L48-L95).

Currently, this multiple toggle is broken on my test site -- although the toggle is enabled in the editing form, the count field is not visible in the editor previews. Different parts of the code obviously convert the string into a boolean differently.